jangaroo-maven/jangaroo-maven-plugin/src/main/java/net/jangaroo/jooc/mvnplugin/WorkspaceConverterMojo.java

```diff
@@ -550,6 +541,59 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     }
   }
 
+  private void writeFile(Path path, InputStream inputStream) throws MojoFailureException {
+    if (inputStream != null) {
+      try (OutputStream outputStream = Files.newOutputStream(path, StandardOpenOption.CREATE_NEW)) {
+        try {
+          byte[] buffer = new byte[8192];
+          int length;
+          while ((length = inputStream.read(buffer)) > 0) {
+            outputStream.write(buffer, 0, length);
+          }
+        } catch (IOException e) {
+          throw new MojoFailureException(e.getMessage(), e.getCause());
+        }
+      } catch (IOException e) {
+        // file already exists, do nothing
+      }
+    }
+  }
+
+  private ByteArrayInputStream createLernaJsonString() throws IOException {
+    Map<String, Object> lernaJson = new LinkedHashMap<>();
+    lernaJson.put("useWorkspaces", true);
+    lernaJson.put("version", "0.0.0");
+    Map<String, Object> lernaCommandMap = new LinkedHashMap<>();
+    Map<String, Object> lernaRunMap = new LinkedHashMap<>();
+    lernaRunMap.put("stream", true);
+    lernaCommandMap.put("run", lernaRunMap);
+    lernaJson.put("command", lernaCommandMap);
+    return new ByteArrayInputStream(objectMapper.writerWithDefaultPrettyPrinter()
+            .writeValueAsString(lernaJson).concat("\n")
+            .getBytes(StandardCharsets.UTF_8));
+  }
+
+  private ByteArrayInputStream createNpmRcString() throws IOException {
+    StringJoiner stringJoiner = new StringJoiner("\n");
+    stringJoiner.add("# as long as we use a typescript beta version we need this");
+    stringJoiner.add("# otherwise the wrong typescript is installed due to a peer dependency");
+    stringJoiner.add("legacy-peer-deps=true");
+    stringJoiner.add("");
+    return new ByteArrayInputStream(stringJoiner.toString().getBytes(StandardCharsets.UTF_8));
+  }
+
+  private ByteArrayInputStream createGitignoreString() throws IOException {
+    StringJoiner stringJoiner = new StringJoiner("\n");
+    stringJoiner.add("# NodeJS");
+    stringJoiner.add("node_modules/");
+    stringJoiner.add("");
+    stringJoiner.add("# Jangaroo Build");
+    stringJoiner.add("dist/");
+    stringJoiner.add("build/");
+    stringJoiner.add("");
+    return new ByteArrayInputStream(stringJoiner.toString().getBytes(StandardCharsets.UTF_8));
+  }
```

Comment:
Maybe we can store these static files as resources?

Reply:
Oh, it already is, so can this be deleted?

Reply:
I forgot to delete them, good you saw it!